src/containers/Network/UpgradeStatus.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from 'react'
+import { useTranslation } from 'react-i18next'
+import axios from 'axios'
+import BarChartVersion from './BarChartVersion'
+import NetworkTabs from './NetworkTabs'
+import Streams from '../shared/components/Streams'
+import Hexagons from './Hexagons'
+import { localizeNumber, FETCH_INTERVAL_MILLIS } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+import Log from '../shared/log'
+
+export const UpgradeStatus = () => {
+  const [vList, setVList] = useState<any>([{}])
+  const [validations, setValidations] = useState([])
+  const [unlCount, setUnlCount] = useState(0)
+  const { t } = useTranslation()
+  const language = useLanguage()
+
+  const aggregateData = (validators: any[]) => {
+    if (!validators) {
+      return []
+    }
+    let total = 0
+    const tempData: any[] = []
+    validators.reduce((aggregate, current) => {
+      const aggregation = { ...aggregate }
+      if (!aggregation[current.server_version]) {
+        aggregation[current.server_version] = {
+          server_version: current.server_version,
+          count: 0,
+        }
+        tempData.push(aggregation[current.server_version])
+      }
+      aggregation[current.server_version].count += 1
+      total += 1
+      return aggregation
+    }, {})
+
+    return tempData
+      .map((item) => ({
+        label: item.server_version ? item.server_version : ' N/A ',
+        value: (item.count * 100) / total,
+        count: item.count,
+      }))
+      .sort((a, b) => (a.label > b.label ? 1 : -1))
```

Comment:
I added a space so that ' N/A ' will always appear left most of the x-axis (valid version should always start with a number).
There can be multiple 'N/A' server as version return null when a server is updating to a new version, based on my observation.